src/gui/touchscreengui.cpp

```diff
@@ -940,6 +1040,23 @@ bool TouchScreenGUI::doubleTapDetection()
 
 }
 
+void TouchScreenGUI::applyJoystickStatus()
+{
+	for (unsigned int i = 0; i < 4; i ++) {
+		SEvent translated;
+		memset(&translated, 0, sizeof(SEvent));
```

Comment:
what ?
if you want to do that properly use C++ initializers

SEvent translated{};

Reply:
Oh, you're right. I just copied that function from another part of the code, seems like it hasn't been changed to C++ initializers.